src/utils.js

```diff
@@ -26,4 +29,23 @@ export function decamelizeKeys (obj) {
 
 export function omitUndefined (obj) {
   return omitBy(obj, isUndefined)
-}
\ No newline at end of file
+}
+
+export function parseWithKey (obj, key) {
+  const parsedObj = attempt(() => JSON.parse(obj))
+  if (!isError(parsedObj)) return parsedObj[key]
+}
+
+export function getLpAuthCookie () {
```

Comment:
Not sure if `utils` is the best place for these methods, as they are pretty specific to the auth functions. LMK if I should move them to a separate file...